content/en/agent/faq/linux-agent-2022-key-rotation.md

```diff
@@ -84,29 +72,28 @@ Run the following commands on the host:
 $ apt-key list
 ```
 
-And check the output for Datadog's key fingerprint `F14F620E` (sometimes a space is included: `F14F 620E`).
+And check the output for Datadog's key fingerprint ending in `F14F620E` or (with a space) `F14F 620E`.
 
 {{% /tab %}}
-{{% tab "RedHat/CentOS" %}}
+{{% tab "RedHat/CentOS/SUSE" %}}
 
 Run the following commands on the host:
 
 ```bash
-$ grep DATADOG_RPM_KEY_FD4BF915.public /etc/yum.repos.d/*datadog*.repo
+$ rpm -q gpg-pubkey-fd4bf915
 ```
 
-And check the output is not empty.
+If the key is trusted, the command has a 0 exit code and outputs:
 
-{{% /tab %}}
-{{% tab "SUSE" %}}
+```
+gpg-pubkey-e09422b3-57744e9e
```

Comment:
Isn't this the old key? For me, the operation actually outputs `gpg-pubkey-fd4bf915-5f573efe`.

Reply:
Indeed.